config/course_information.yml

```diff
@@ -0,0 +1,18 @@
+shared:
+  where_you_will_train:
+    universities:
+      except_provider_codes:
+        - B31
```

Comment:
@JR-G do you know if there are plans to remove exceptions for specific providers in the codebase?

Reply:
Personally I would rather we just remove all of them and make no exceptions for any of them – apart from the UCL one, since that is a different situation. 

However, happy to move ahead with this as an interim solution, if we can reach out to the two providers to understand why we had this in place.

Reply:
I agree

Reply:
Yeah, B31 is not a university so this is not technically relevant. I'm just reproducing what existed before.
This should just apply to all universities.